docker/openqa_data/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM opensuse:13.2
+MAINTAINER Jan Sedlak <jsedlak@redhat.com> Josef Skladanka <jskladan@redhat.com>
+
+RUN zypper --non-interactive --gpg-auto-import-keys in ca-certificates git wget
```

Comment:
--gpg-auto-import-keys disables a major feature of the package manager. I don't  think you want to do that for production.